integration_tests/pages/barcode/printCoversheets.ts

```diff
@@ -1,16 +1,16 @@
 /* eslint-disable import/no-cycle */
 import Page from '../page'
 import SelectEnvelopeSizePage from './selectEnvelopeSize'
+import FindRecipientByPrisonNumberPage from './findRecipientByPrisonNumber'
 
 export default class PrintCoversheetsPage extends Page {
   constructor() {
-    super('print-coversheets', { axeTest: false })
```

Comment:
Now there is a 1 second delay before JS tries to auto-download the PDF we can remove this config and comment and allow nice things like axe testing to work 👍